geode-benchmarks/src/main/java/org/apache/geode/benchmark/tests/GeodeBenchmark.java

```diff
@@ -45,11 +46,20 @@ public static TestConfig createConfig() {
     config.durationSeconds(BENCHMARK_DURATION);
     config.threads(THREADS);
 
-    final String sniProp = System.getProperty("withSniProxy");
-    final boolean doSni = sniProp != null && !sniProp.equals("false");
+    String prop = System.getProperty("withSniProxy");
```

Comment:
`Boolean.getBoolean(name)` returns false if the property is missing or exists but is not "true".

Reply:
I thought this code was curious, too.  I'll ask Bill about it because this pattern was already in place.

Reply:
I simply didn't know about `getBoolean()`. `GeodeProperties.isPropertySet(String)` can be replaced with `getBoolean()` too.

Reply:
Hmm, that's as may be but the code you wrote doesn't do the same thing as Boolean.getBoolean().  The code allows the property to be set with a value that, if not "false", enables the setting.  Boolean.getBoolean() returns `false` if the property is not set or if it is set but is not exactly equal to "false". 
Changing the code to use Boolean.getBoolean() causes your test GeodeBenchmarkTest.withSniProxyNotLowercaseFalse to fail, so I'd like some assurance that it's okay to delete that test.